src/run.php

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env php
+<?php
+// src/run.php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+
+$setAutoloader();
+$terminus = new Terminus();
+$command_factory = new AnnotatedCommandFactory();
+//$formatter_manager = new FormatterManager();
+
+$discovery = new CommandFileDiscovery();
+$command_files = $discovery->discover(__DIR__, '\Commands');
+foreach ($command_files as $command_class) {
+  $command_instance = new $command_class();
+  //$command_factory->commandProcessor()->setFormatterManager($formatter_manager);
+  $command_list = $command_factory->createCommandsFromClass($command_instance);
+  foreach ($command_list as $command) {
+    $terminus->add($command);
+  }
+}
+
+/**
+  * Sets the autoloader up to retrieve files as necessary
+  *
+  * @return void
+  */
+function setAutoloader() {
```

Comment:
@TeslaDethray You should create a 0.x branch for the old composer, and fix up the composer.json of the new composer to be the way you'd like it to be. I think things are far enough along to fork. Only minor changes, if any, will be going in to the old composer branch at this point, right?